.circleci/config.yml

```diff
@@ -17,25 +17,31 @@ jobs:
           name: Linter
           command: python3 -m black --check ./
   pytest:
-    machine: true
+    machine:
+      image: ubuntu-2004:202201-02
+      docker_layer_caching: true
```

Comment:
This one works great! Major improvement!

<img width="1093" alt="Screenshot 2022-06-25 at 17 04 22" src="https://user-images.githubusercontent.com/3535056/175779431-c791feb4-5e1e-417f-838c-c6d09776f34b.png">

Reply:
@rodriguesk don't need to remove it, it works fine for the `pytest` step